lib/recurly/client.rb

```diff
@@ -76,8 +79,16 @@ def get(path, **options)
       path = scope_by_site(path, **options)
       request = Net::HTTP::Get.new path
       set_headers(request, options[:headers])
-      http_response = run_request(request, options)
-      handle_response! request, http_response
+
+      retries = 0
```

Comment:
What do you think about moving this logic to `run_request` or find a way to share it? `next_page` needs it for instance as that's what the Pager uses. It'll make it easier to turn on for other methods in the future as well.

Reply:
Valid! I found a way to move it to `#run_request` that doesn't add much complexity.